src/components/LabelComponent/LabelComponent.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react"; 
```

Comment:
sorry, not sure where this component gets used...?

Reply:
I'm still not 100% sure on what extending component does. I've been just putting that on all my components lol